packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -311,6 +305,17 @@ export default class AccountTransferStore {
 
                 return { ...account, ...found_account, account_type: CFD_PLATFORMS.MT5 };
             }
+            if (
+                account.account_type === CFD_PLATFORMS.CTRADER &&
```

Comment:
Here TS error, because we are missing `'ctrader'` value in account_type property in deriv/api-type, after the package will be updated, the error will go away.